ada_feeding_perception/ada_feeding_perception/food_on_fork_logistic_reg_training.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+import joblib
+
+
+def main():
+    csv_to_read_from = "../datasets/Ross_7-11-23.csv"
```

Comment:
Why is this called `Ross_...`? Seems like a very unclear name -- please change it to something more descriptive.